src/aoptk/scispacy_find_chemical.py

```diff
@@ -0,0 +1,23 @@
+from aoptk.find_chemical import FindChemical
+from aoptk.chemical import Chemical
+import spacy
+import scispacy
+
+class ScispacyFindChemical(FindChemical):
+    def __init__(self, sentences: list[str]):
+        self._sentences = sentences
+
+    def find_chemical(self) -> list[Chemical]:
+        chem_list = []
+        for sentence in self._sentences:
+            chem = self.scispacy_find_chemical(sentence)
+            chem_list.append(chem)
+        return chem_list
```

Comment:
This is just a wrapper, which becomes not needed if you change the interface function accordingly. The 'scispacy_find_chemical' then becomes the 'find_chemicals' function.